MaterialSkinExample/MainForm.cs

```diff
@@ -22,9 +23,9 @@ public MainForm()
             materialSkinManager.EnforceBackcolorOnAllComponents = true;
 
             // MaterialSkinManager properties
-            materialSkinManager.AddFormToManage(this);
-            materialSkinManager.Theme = MaterialSkinManager.Themes.LIGHT;
-            materialSkinManager.ColorScheme = new ColorScheme(Primary.Indigo500, Primary.Indigo700, Primary.Indigo100, Accent.Pink200, TextShade.WHITE);
+            materialSkinManager.AddFormToManage(this);
+			materialSkinManager.SetThemes(new ThemeLight(materialSkinManager), new ThemeDark(materialSkinManager), true);

```

Comment:
This is a breaking change, can we keep the `MaterialSkinManager.Themes.LIGHT` somehow? Maybe use a static class? also please fix the tab spacing :)